trie/triedb/pathdb/history.go

```diff
@@ -446,12 +457,13 @@ func (r *decoder) readStorage(accIndex accountIndex) ([]common.Hash, map[common.
 }
 
 // decode deserializes the account and storage data from the provided byte stream.
-func (h *history) decode(accountData, storageData, accountIndexes, storageIndexes []byte) error {
+func (h *history) decode(accountData, storageData, accountIndexes, storageIndexes, trieNodes []byte) error {
 	var (
 		accounts    = make(map[common.Address][]byte)
 		storages    = make(map[common.Address]map[common.Hash][]byte)
 		accountList []common.Address
 		storageList = make(map[common.Address][]common.Hash)
+		encoded     []journalNodes
```

Comment:
Polish this var name to keep a reasonable semantics.

Reply:
Fixed